File: c/1295.c
int findNumbers(int* n, int l) {
    int c = 0;
    for (int i = 0; i < l; i++)
        if ((n[i] > 9 && n[i] < 100) || (n[i] > 999 && n[i] < 10000) ||
            n[i] == 100000)
            c++;
    return c;
}

File: javascript/1295.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function (nums) {
    let c = 0;
    for (let i of nums)
        if ((i > 9 && i < 100) || (i > 999 & i < 10000) || i == 100000)
            c++;
    return c;
};
